### ISSUE TITLE
Is there any way to know slide state by gesture?

### ISSUE THREAD

**Issue by Woongsik** [nativescript-community/ui-drawer#19]
Hi, thanks for the great plugin!

I wonder if there is a way to know about slide state by gesture... 
I know the this.$refs.drawer.isOpened() manually but I am looking for a way to get it by gesture and emit from the component.
Since the component uses swipe gesture ... 

ie. <Drawer @leftSlideState="(open or close) => {}"> 
Any workaround is also welcome...

**Comment by farfromrefug** [comment]
Yes should be doable. Can't do it right now. Would you want to create a PR?

**Comment by Woongsik** [comment]
Hi, I can try it. Could you give me a quick instruction that I should look into?

**Comment by farfromrefug** [comment]
@Woongsik the repo readme has instructions on how to use it. As for the code you could simply emit where you want to notify of state change.
